### ISSUE TITLE
Map.PanLock does not prevent/disable fling

### ISSUE THREAD

**Issue by RudyTheDev** [Mapsui/Mapsui#1679]
Hello,

Setting `MapControl.Map.PanLock = true;` does not prevent/disable the map from flinging animation when swiping.

**To Reproduce**

New empty NET 6.0 Windows app.

XAML:

`<wpf:MapControl Name="MyMapControl" />`

In window ctor:

```
MyMapControl.Map = new Map();
MyMapControl.Map.Layers.Add(OpenStreetMap.CreateTileLayer());
MyMapControl.Map.PanLock = true;
```

Run app. Attempt to pan. No panning as expected. Attempt to fling. Observe map flinged.

![mapsui fling](https://user-images.githubusercontent.com/3857299/180602011-2bb1b065-b7cf-4260-86c5-b25c82b57fef.gif)

(Note that for the GIF I changed mouse cursor for mouse down/up events so it's clear when the mouse is down.)

Using Mapsui.Wpf 4.0.0-beta.5.

I did not see a separate option to disable fling only, so I am assuming this should also be disabling flinging.

After some experimentation I see that `MapControl.OnFlinged` checks `if (e.Handled) return true;`, so I technically can do:

```MyMapControl.Fling += MyMapControlOnFling;```

and

```
private void MyMapControlOnFling(object? sender, SwipedEventArgs e)
{
    e.Handled = true;
    // or generally, e.Handled = MyMapControl.Map.PanLock;
}
```

Which does prevent flinging, but seems like a roundabout way of doing this by "manually" consuming the fling event. I am not sure if this is by design.

**Comment by pauldendulk** [comment]
Thanks for you clear bug report. I can also reproduce it in the PanLock samples (under the Navigation category) when running Mapsui.Samples.Wpf. PanLock should indeed also prevent fling so this is a bug.

**Comment by pauldendulk** [comment]
This is fixed and will be released in 4.0.0-beta.9.
